tests/unit/sparql/buildQuery.spec.ts

```diff
@@ -1,14 +1,51 @@
 import buildQuery from '@/sparql/buildQuery';
+import PropertyValueRelation from '@/data-model/PropertyValueRelation';
 
 describe( 'buildQuery', () => {
 
 	it( 'builds a query from a property and a string value', () => {
 		const propertyId = 'P666';
 		const value = 'blah';
+		const propertyValueRelation = PropertyValueRelation.Matching;
 		expect( buildQuery( { condition: {
 			propertyId,
 			value,
+			propertyValueRelation,
 		} } ) ).toEqual( `SELECT ?item WHERE { ?item (p:${propertyId}/ps:${propertyId}) "${value}". }` );
 	} );
 
+	it( 'builds a query from a property and a string value with not matching selected', () => {
+		const propertyId = 'P666';
+		const value = 'blah';
+		const propertyValueRelation = PropertyValueRelation.NotMatching;
+
+		/*
+		* adding an array here to circumvent backticks + indentation problems between
+		* the expected and received strings
+		*/
+		const expectedQuery = [
+			'SELECT ?item WHERE {',
+			`  ?item (p:${propertyId}/ps:${propertyId}) ?instance.`,
+			`  FILTER(?instance != "${value}")`,
+			'}',
+		].join( '\n' );
+
+		expect( buildQuery( { condition: {
+			propertyId,
+			value,
+			propertyValueRelation,
+		} } ) ).toEqual( expectedQuery );
+	} );
+
+	it( 'builds a query from a property with "any value" selected', () => {
+		const propertyId = 'P666';
+		const propertyValueRelation = PropertyValueRelation.Regardless;
+
+		expect( buildQuery( { condition: {
+			propertyId,
+			value: '',
```

Comment:
I first tried not sending the value but it got messy with the types from the Terms tripleObject in `src/sparql/QueryObjectBuilder.ts `. Happy to read suggestions.

Reply:
Mh, I think it is okay to use it like that for now. Maybe we could also use `null`, but an empty string is also just fine here as that should not be possible by user input.